src/component-library/Meter/Meter.style.tsx

```diff
@@ -1,13 +1,61 @@
 import styled from 'styled-components';
 
+import { Span } from '../Text';
 import { theme } from '../theme';
 
-const StyledWrapper = styled.div`
-  padding-top: 8px;
+type StyledWrapperProps = {
+  $showRanges?: boolean;
+};
+
+const StyledWrapper = styled.div<StyledWrapperProps>`
+  position: relative;
+  padding-top: ${({ $showRanges }) => ($showRanges ? theme.spacing.spacing8 : '8px')};
   padding-bottom: 36px;
   width: 100%;
 `;
 
+type StyledIndicatorProps = {
```

Comment:
Very minor and definitely not a blocker, but I think I'd prefer it if types were grouped together at the top of the file rather than preceding the component. I can see why it makes sense to do it in the way that you have, but I think it would be a bit more consistent with the rest of the app if we grouped all of the types together.